### PR TITLE
Stations Dashboard

### PR DESCRIPTION
**Pull request by pollockcl**
SUBMITTED BY: pollockcl

Major Changes
  * Added stations/dashboard controller, routes, namespace
  * added view for the dashboard
  * model and feature tests

Applicable Bug Fixes
  *
  *
  *

Passing Tests? (y)

REVIEWED BY:

### CODE REVIEW

File: app/controllers/stations/dashboard_controller.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+module Stations
+  class DashboardController < ApplicationController
+    def index
```
Comment (pollockcl) [inline app/controllers/stations/dashboard_controller.rb]: I need to figure out how to get presenters working to get rid of all these instance variables. If you know of a better way in the interim then let me know and we can fix it.

Comment (alexcutschall) [reply app/controllers/stations/dashboard_controller.rb]: Hmm, I'm not too sure on presenters. The instance variables work for the time being

File: app/models/station.rb [hunk 2]
```diff
@@ -5,6 +5,34 @@ class Station < ApplicationRecord
...
+
+  def self.least_docks
+    find_by(dock_count: minimum(:dock_count))
+  end
+
+  def self.min_docks
+    minimum(:dock_count)
+  end
+
+  def self.max_docks
```
Comment (alexcutschall) [inline app/models/station.rb]: All of these look great!

File: app/views/stations/dashboard/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<%= @total_stations %>
+<%= @average_bikes %>
+<%= @most_docks.name %>: <%= @min_docks %>
+<%= @least_docks.name %>: <%= @max_docks %>
```
Comment (alexcutschall) [inline app/views/stations/dashboard/index.html.erb]: I'll go ahead and style this now

File: db/schema.rb [hunk 4]
```diff
@@ -18,9 +18,13 @@
   create_table "accessories", force: :cascade do |t|
     t.string "title"
     t.decimal "price"
+<<<<<<< HEAD
     t.text "description"
     t.string "image", default: "https://i1.wp.com/thebeautyinyou.mlethadaniels.net/wp-content/uploads/2017/10/Smart-Indoor-Recumbent-Exercise-Bike-JOROTO-MH20-Cardio-Fitness-Cycling-Machine-Home-Stationary-Trainer-with-Pulse-0.jpg?resize=150%2C150"
     t.string "status"
+=======
+    t.string "image"
+>>>>>>> added stations dashboard namespace, controller, view and tests
```
Comment (alexcutschall) [inline db/schema.rb]: Looks like there's a clash here?

**Pull request merged**